app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -255,6 +255,16 @@ def add_ep_merge_special_issue!
     update!(form_json: JSON.dump(form))
   end
 
+  def conditionally_determine_ep_merge
+    actor = OpenStruct.new({ flipper_id: user_uuid })
+    feature_enabled = Flipper.enabled?(:disability_526_ep_merge_api, actor)
+    if Flipper.enabled?(:disability_compensation_production_tester, actor)
+      feature_enabled = false
+      Rails.logger.info("EP merge skipped for submission #{id}, user_uuid #{user_uuid}")
+    end
+    feature_enabled
```

Comment:
I think this method could read a bit smoother if you treat the production tester as a special-case override.

```suggestion
    if Flipper.enabled?(:disability_compensation_production_tester, actor)
      Rails.logger.info("EP merge skipped for submission #{id}, user_uuid #{user_uuid}")
      return false
    end
    Flipper.enabled?(:disability_526_ep_merge_api, actor)
```